docs/resources/aws_iam_users.md

```diff
@@ -38,8 +38,26 @@ The following examples show how to use this InSpec audit resource.
       it { should exist }
     end
 
+### Test that all users that have a console password have Multi-Factor Authentication enabled
+
+    describe aws_iam_users.where(has_console_password?: true) do
```

Comment:
Just a note, I made #103 to document `where` and the various things you can pass to it.  We'll address in a future PR.